releasenotes/notes/improve-collect-cliffords-f57aeafe95460b18.yaml

```diff
@@ -0,0 +1,47 @@
+---
+upgrade:
+  - |
+    Upgrading :class:`~CollectCliffords` transpiler pass to collect and combine blocks
+    of "clifford gates" into :class:`qiskit.quantum_info.Clifford` objects, where the
+    "clifford gates" may now also include objects of type :class:`.LinearFunction`,
+    :class:`qiskit.quantum_info.Clifford`, and :class:`~qiskit.circuit.library.PauliGate`.
```

Comment:
We usually use `upgrade` for API changes, where users will actively need to adjust their code to handle something we do differently now.  I may misunderstand a little, but I think this is more like a new feature - the `CollectCliffords` pass is now just more efficient, and can collect more objects into a single `Clifford`.

Reply:
Thanks. I have indeed misunderstood what `update` means, fixed in [d9d9c17](https://github.com/Qiskit/qiskit-terra/pull/9169/commits/d9d9c178aaf952905e8ddcc0c5f511f785cb889e).